dozer-sql/src/pipeline/mod.rs

```diff
@@ -13,3 +13,16 @@ mod window;
 
 #[cfg(test)]
 mod tests;
+
+#[cfg(feature = "onnx")]
+pub mod udfs_errors;
```

Comment:
Rename to `onnx` and put `DozerSession` into it?

Reply:
ok